### DISCUSSION TITLE
licence for imagemagick software redistribution

### DISCUSSION THREAD

**Discussion by SylvainArd** [ImageMagick/ImageMagick#6115] [General]
Hello,
I have done a color management program wich use imagemagick. I included imagemagick in my installer but I saw in the licence that we must have agreement from you to do this isn't it ?
Thank you
Best regards

**Comment by urban-warrior** [comment]
Have you read the  ImageMagick license @ https://imagemagick.org/script/license.php?

**Comment by SylvainArd** [comment]
Yes



Envoyé de mon iPhone

**Comment by SylvainArd** [comment]
I read that we must put a copy of the im license in our program and have im authorization to redistribute it

Envoyé de mon iPhone

**Reply by fmw42** [reply]
Please identify the URL where that is stated.  Are you sure you are on the actual Imagemagick web site and not some other web site?

**Comment by SylvainArd** [comment]
yes I am sure

https://imagemagick.org/script/license.php

"redistribute any piece of ImageMagick-originated software without proper
attribution;"
Sylvain Ard

0549507724
0778380991
***@***.***
http://sylvain-ard.fr
Entreprise individuelle SIRET : 80079243400022
Appt 26 Bât A Résidence Le Patio
83 rue de la Bugellerie
86000 Poitiers


Le mer. 1 mars 2023 à 18:02, Fred Weinhaus ***@***.***> a
écrit :

**Reply by urban-warrior** [reply]
The agreement is to meet the requirements of the license to utilize ImageMagick.  There is no requirement that you need an agreement with the ImageMagick team.   The license is designed to prevent folks from utilizing ImageMagick and claiming they wrote it.  In your distribution, its sufficient to include a link to the ImageMagick license which alerts your user base that your app includes some portion of ImageMagick.

**Comment by SylvainArd** [comment]
OK thank you I have done that
Sylvain Ard
0549507724
0778380991
***@***.***
http://sylvain-ard.fr
Entreprise individuelle SIRET : 80079243400022
Appt 26 Bât A Résidence Le Patio
83 rue de la Bugellerie
86000 Poitiers
